src/pprint/js_pprint.cc

```diff
@@ -684,6 +684,7 @@ counted_t<const document_t> render_as_javascript(const ql::raw_term_t &t) {
 // - `should_continue_string` should add the new Term to the list if
 //   it should be used in a string-of-dotted-expressions context.  So
 //   like `r.foo(1).bar(2).baz(4)` instead of `r.eq(1, 2)`.
+//   ^^ What does this mean??
```

Comment:
I think it's referring to cases where you'd naively pretty-print `r.eq(var_0, 1)`, but you'd prefer `var_0.eq(1)`. The inverse example would be that you don't want `r.expr('foo').db()`, it should be `r.db('foo')`.